pkg/quota/evaluator/core/persistent_volume_claims.go

```diff
@@ -144,7 +144,7 @@ func (p *pvcEvaluator) GroupKind() schema.GroupKind {
 // Handles returns true if the evaluator should handle the specified operation.
 func (p *pvcEvaluator) Handles(a admission.Attributes) bool {
 	op := a.GetOperation()
-	if op == admission.Create {
+	if op == admission.Create || op == admission.Update {
```

Comment:
open an issue for a follow-on to ensure that on update, we change the max(new,old) per resource.

Reply:
yep done here - https://github.com/kubernetes/kubernetes/issues/51736

Reply:
should only handle update if the feature gate is enabled... otherwise, you're opening PVs to negative quota spoofing on update

Reply:
actually, this already interacts oddly with the uninitialized bit below

Reply:
added feature gate check